docs/book/src/tasks/installation.md

```diff
@@ -11,7 +11,7 @@ Cluster API requires an existing kubernetes cluster accessible via kubectl, choo
 
 1. **Kind**
 
-{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere"}}
+{{#tabs name:"kind-cluster" tabs:"AWS,Docker,vSphere,OpenStack"}}
```

Comment:
It seems that we're duplicating instructions for all except Docker? If so, we should probably collapse them into one

Reply:
Good idea. Done.

Reply:
Looking good.

/lgtm